README.md

```diff
@@ -167,54 +168,48 @@ resource-key = uri / http-headers
 http-headers = bstr
 ```
 
+A `uri` `resource-key` is equivalent to an `http-headers` block with ":method"
+set to "GET" and with ":scheme", ":authority", and ":path" headers set from the
```

Comment:
Later the resource key is described with a particular order, should that be reflected here (or pulled out and referenced in both places)